src/manifest.json

```
@@ -0,0 +1,22 @@
+{
+  "short_name": "quiver",
+  "name": "quiver",
+  "description": "a modern commutative diagram editor",
+  "start_url": ".",
+  "scope": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
```

```suggestion
  "theme_color": "#1a1a1a",
```
This is the colour used in the **quiver** logo.